test_e2e/test_app_selenium.py

```diff
@@ -29,16 +45,20 @@ def create_trello_board():
 def delete_trello_board(trello_board_id):
     params = {"key": os.getenv("TRELLO_KEY"), "token": os.getenv("TRELLO_TOKEN")}
     requests.delete("https://api.trello.com/1/boards/" + trello_board_id, data = params)
-
+'''
 @pytest.fixture(scope = 'module')
 def test_app():
     #Create the new board and update the board id enviroment variable 
 
     file_path = find_dotenv('.env')
     load_dotenv(file_path, override=True)
+    mongo_client = pymongo.MongoClient(os.getenv("MONGO_CLIENT"))
+
+    database_name = "Temporary Database"
+    create_database(mongo_client, database_name)
```

Comment:
Thought this was how it was done re. connecting to the database.

Also with regards to the creation of an item, I assume you mean move lines from the `create_database` function to `test_app` . But how are we to verify that the name of the temp database is in the list of database names otherwise. What should the `create_database` be returning?

Reply:
So just try to create and remove an item from the database then?